README.md

```diff
@@ -1,4 +1,44 @@
-# CorsixTH Community Content
-Fan levels and campaigns
+# CorsixTH custom campaigns, levels and maps
 
-<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```

Comment:
Could you restore the exact code for this line. Changing to "All works" is fine. A copyright statement should not float all the way to the bottom of the file.